scripts/vote_2023_10_03.py

```diff
@@ -0,0 +1,172 @@
+"""
+Voting 03/10/2023.
+
+1. Add node operator A41 with reward address `0x2A64944eBFaFF8b6A0d07B222D3d83ac29c241a7`
+2. Add node operator Develp GmbH with reward address `0x0a6a0b60fFeF196113b3530781df6e747DdC565e`
+3. Add node operator Ebunker with reward address `0x2A2245d1f47430b9f60adCFC63D158021E80A728`
+4. Add node operator Gateway.fm AS with reward address `0x78CEE97C23560279909c0215e084dB293F036774`
+5. Add node operator Numic with reward address `0x0209a89b6d9F707c14eB6cD4C3Fb519280a7E1AC`
+6. Add node operator ParaFi Technologies LLC with reward address `0x5Ee590eFfdf9456d5666002fBa05fbA8C3752CB7`
+7. Add node operator RockawayX Infra with reward address `0xcA6817DAb36850D58375A10c78703CE49d41D25a`
+8. Grant STAKING_MODULE_MANAGE_ROLE to Lido Agent
+9. Set Jump Crypto targetValidatorsLimits to 0
+10. Update Anchor Vault implementation from `0x07BE9BB2B1789b8F5B2f9345F18378A8B036A171` to TBA
```

Comment:
Please incorporate the following implementation address: https://etherscan.io/address/0x9530708033E7262bD7c005d0e0D47D8A9184277d
The code corresponds to the following commit: https://github.com/lidofinance/anchor-collateral-steth/blob/c186bf3bb89f2945646a76a4e033030c4ea2842a/contracts/AnchorVault.vy